apis/meta/annotations.go

```diff
@@ -21,19 +21,19 @@ const (
 	// outside of the defined schedule. Despite the name, the value is not
 	// interpreted as a timestamp, and any change in value shall trigger a
 	// reconciliation.
-	// DEPRECATED: has been replaced by ReconcileRequestAnnotation.
+	// DEPRECATED: has been replaced by ReconcileRequestAnnotation. For
```

Comment:
I think it's time to remove this, we've switched all controllers to `ReconcileRequestAnnotation` in 2020.

Reply:
I would probably do this in a separate PR to not incorporate more "hidden" breaking changes in this one.